File: TLX-Toki/Pascal/Operasi.pas
var
	A, B : integer;
	C : real;
	Q : char;
begin
	writeln('Masukan bilangan A dan B:');
	readln(A,B);
	writeln('Masukan pilihan operasi :');
	readln(Q);

	case q of
		'a' : writeln('A + B = ', A + B);
		'b' : writeln('A - B = ', A - B);
		'c' : writeln('A * B = ', A * B);
		'd' : begin
			C := A * 1.00 / B;
			writeln('A / B = ',C);
			end;
		'e' : writeln('A div B = ', A div B);
		'f' : writeln('A mod B = ', A mod B);
		else 
			writeln('Bukan pilihan menu yang benar');
	end;
end.

File: TLX-Toki/CPP/operasi.cpp
#include <iostream>
using namespace std;

int main() {
	int A, B;
	float C;
	char q;

	cout << "Masukan bilangan A dan B:" << endl;
	cin>>A>>B;
	cout << "Masukan pilihan operasi :" << endl;
	cin>>q;

	switch(q){
		case 'a' : cout << "A + B = " << A + B; break;
		case 'b' : cout << "A - B = " << A - B; break;
		case 'c' : cout << "A * B = " << A * B; break;
		case 'd' : {
			C = A * 1.00 / B;
			cout << "A / B = ";
			cout << C; break;}
		case 'e' : cout << "A div B = " << A / B; break;
		case 'f' : cout << "A mod B = " << A % B; break;
		default : cout << "Bukan pilihan menu yang benar";
	}
}